brave/src/main/java/brave/propagation/TraceContextHolder.java

```diff
@@ -0,0 +1,49 @@
+package brave.propagation;
+
+public class TraceContextHolder {
```

Comment:
My personal opinion is that we should avoid a class like this that uses statics to keep state. If there was a Factory in between the implementation and the users that would be more desirable to me